delfin/api/v1/storages.py

```diff
@@ -192,6 +192,27 @@ def _storage_exist(self, context, access_info):
 
         return False
 
+    def perf_collect(self, storage_id, interval, is_historic, resource):
```

Comment:
Should we keep it in 'StorageController' ? How about having a 'PerformanceController' ? Idea is to have all registration, query, implementaion in this class ..

Reply:
ok. i will seprate the file and classes